src/lib/list/list.ts

```diff
@@ -46,6 +46,23 @@ export const _MatListItemMixinBase = mixinDisableRipple(MatListItemBase);
 })
 export class MatNavList extends _MatListMixinBase implements CanDisableRipple {}
 
+
+@Component({
+  moduleId: module.id,
+  selector: 'mat-button-list',
+  exportAs: 'matButtonList',
+  host: {
+    'role': 'button list',
+    'class': 'mat-button-list'
+  },
+  templateUrl: 'list.html',
+  styleUrls: ['list.css'],
+  inputs: ['disableRipple'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MatButtonList extends _MatListMixinBase implements CanDisableRipple {}
```

Comment:
Since it looks like `.mat-button-list` doesn't differ from `.mat-list` for any styles or behavior, you should be able to just add `mat-action-list` to the existing selector for `MatList` without creating a new component